pkg/api/validation/validation.go

```diff
@@ -1760,6 +1760,9 @@ func ValidateVolumeMounts(mounts []api.VolumeMount, volumes sets.String, fldPath
 		if mountpoints.Has(mnt.MountPath) {
 			allErrs = append(allErrs, field.Invalid(idxPath.Child("mountPath"), mnt.MountPath, "must be unique"))
 		}
+		if !path.IsAbs(mnt.MountPath) {
```

Comment:
@sttts From [the Docker Official Doc](https://docs.docker.com/engine/tutorials/dockervolumes/#mount-a-host-directory-as-a-data-volume),

> The `container-dir` must always be an **absolute path** such as /src/docs.

So the container cannot be started up if you use a non-absolute path for `mnt.MountPath`, just as what was reported in #48749.